### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by asthaaaa**
Add Alert when indices are deleted
Add "." and "/" to log lines

### CODE REVIEW

File: src/main/webapp/servlet.js [hunk 1]
```diff
@@ -56,5 +56,5 @@ const deleteIndices = async() => {
     await fetch(DELETE, {
         method: POST
 });
-
+    alert("Data Deleted");
```
Comment (shudhatmajain) [inline src/main/webapp/servlet.js:59]: Ideally, we should alert before deleting.

File: src/test/java/com/google/error_analyzer/backend/StoreLogHelperTest.java [hunk 2]
```diff
@@ -52,7 +52,7 @@ public void convertToJsonStringTest() throws IOException {
     public void cleanLogText_removeSpecialCharacters() {
         String actual =
             storeLogHelper.cleanLogText("Error1:\"^&index not found?/,*\"");
-        String expected = new String("Error1:\" index not found \"");
+        String expected = new String("Error1:\" index not found / \"");
```
Comment (shudhatmajain) [inline src/test/java/com/google/error_analyzer/backend/StoreLogHelperTest.java:55]: Test "." as well.

File: src/main/webapp/servlet.js (hunk 1 above)
Comment (asthaaaa) [reply src/main/webapp/servlet.js:59]: We are confirming with the user before deleting

Comment (nishantnahata) [inline src/main/webapp/servlet.js:59]: Something like "Log files deleted"?

**Pull request merged**